101-vm-simple-linux/flash-vm-linux.json

```diff
@@ -0,0 +1,158 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
@geo1211 what is the purpose of this file? This seems too simple plus there is already a template here